src/libs/actions/Report.js

```diff
@@ -369,6 +370,7 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
     const params = {
         reportID,
         emailList: participantList ? participantList.join(',') : '',
+        parentReportActionID: parentReportActionID === '0' ? null : parentReportActionID,
```

Comment:
you can just do,
```
parentReportActionID,
```
over here.

to the backend providing parentReportActionID as `''`, `null`, `'0'` would be just the same and its converted to `0` for these values.